common/luaUtilities/serpent.lua

```diff
@@ -0,0 +1,152 @@
+local n, v = "serpent", "0.303" -- (C) 2012-18 Paul Kulchenko; MIT License
```

Comment:
Serpent allows serialization/deserialization of data and functions with a consistent interface. It saves a lot of effort here, since all synced function calls need to be serialized/deserialized.

Reply:
```suggestion
-- provides a consistent interface for de/serialisation
local n, v = "serpent", "0.303" -- (C) 2012-18 Paul Kulchenko; MIT License
```